apps/deploy-web/env/.env.production

```
@@ -35,4 +35,5 @@ NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN=*:production.3de2776029292b1860a520b1aa3e
 NEXT_PUBLIC_GTM_ID=GTM-MSF384N3
 
 NEXT_PUBLIC_STRIPE_PUBLISHABLE_KEY=pk_live_ytcG6PkH6KnmGR4Ge83ApYGr004iygcjjJ
-NEXT_PUBLIC_TRIAL_CREDITS_AMOUNT=1
\ No newline at end of file
+NEXT_PUBLIC_TRIAL_CREDITS_AMOUNT=1
+NEXT_PUBLIC_MANAGED_WALLET_TRIAL_BLOCKED_GPU_MODELS=nvidia/b300,nvidia/b200,nvidia/h200,nvidia/h100,nvidia/pro6000se,nvidia/pro6000we,nvidia/a100,nvidia/rtx5090,nvidia/rtx5090m,nvidia/rtx4090,nvidia/rtx3090
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)